mlir/test/Target/LLVMIR/Import/debug-info-records.ll

```diff
@@ -0,0 +1,92 @@
+; RUN: mlir-translate -import-llvm -mlir-print-debuginfo -convert-debug-rec-to-intrinsics -emit-expensive-warnings -split-input-file %s 2>&1 | FileCheck %s
+; RUN: mlir-translate -import-llvm -mlir-print-debuginfo -emit-expensive-warnings -split-input-file %s 2>&1 | FileCheck %s
+
+; CHECK: @callee()
+define void @callee() {
+  ret void
+}
+
+define void @func_with_empty_named_info() {
+  call void @callee()
+  ret void
+}
+
+define void @func_no_debug() {
+  ret void
+}
+
+; CHECK: llvm.func @func_with_debug(%[[ARG0:.*]]: i64
+define void @func_with_debug(i64 %0) !dbg !3 {
+
+  ; CHECK: llvm.intr.dbg.value #di_local_variable = %[[ARG0]] : i64
+  ; CHECK: llvm.intr.dbg.value #di_local_variable1 #llvm.di_expression<[DW_OP_LLVM_fragment(0, 1)]> = %[[ARG0]] : i64
+  ; CHECK: %[[CST:.*]] = llvm.mlir.constant(1 : i32) : i32
+  ; CHECK: %[[ADDR:.*]] = llvm.alloca %[[CST]] x i64
+  ; CHECK: llvm.intr.dbg.declare #di_local_variable2 #llvm.di_expression<[DW_OP_deref, DW_OP_LLVM_convert(4, DW_ATE_signed)]> = %[[ADDR]] : !llvm.ptr
```

Comment:
```suggestion
  ; CHECK: llvm.intr.dbg.value #di_local_variable{{.*}} = %[[ARG0]] : i64
  ; CHECK: llvm.intr.dbg.value #di_local_variable{{.*}} #llvm.di_expression<[DW_OP_LLVM_fragment(0, 1)]> = %[[ARG0]] : i64
  ; CHECK: %[[CST:.*]] = llvm.mlir.constant(1 : i32) : i32
  ; CHECK: %[[ADDR:.*]] = llvm.alloca %[[CST]] x i64
  ; CHECK: llvm.intr.dbg.declare #di_local_variable{{.*}} #llvm.di_expression<[DW_OP_deref, DW_OP_LLVM_convert(4, DW_ATE_signed)]> = %[[ADDR]] : !llvm.ptr
```
nit: let's maybe not match the exact numbers here. Who knows if this ordering ever changes. Alternatively matching the aliases may also be a good option.